test/stringClassSpec.js

```diff
@@ -74,11 +76,19 @@ describe('To Currency', () => {
   it('should return string in currency format', () => {
     expect('20003.67'.toCurrency()).toEqual('20,003.67');
   });
+
+  it('should return string in currency format with decimal', () => {
+    expect('20003'.toCurrency()).toEqual('20,003.00');
+  });
 });
 
 describe('From Currency', () => {
   it('should return string in number format', () => {
-    expect('20,003.67'.toCurrency()).toEqual('20003.67');
+    expect('20,003.67'.fromCurrency()).toEqual('20003.67');
+  });
+
+  it('should return a number format without insignificant zero', () => {
+    expect('20,003.00'.fromCurrency()).toEqual('20003');
   });
 });
 
```

Comment:
For `fromCurrency` and `toCurrency`, it would be useful to test with strings that are well formed currencies e.g having letters or misplaced commas and ensure that the implementation handles those edge cases.